samples/bluetooth/beacon/Kconfig

```diff
@@ -1,8 +0,0 @@
-# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
Instead of adding a `[noup]` patch removing files introduced with a `[temphack]` patch, then please revert the `[temphack]` commit (https://github.com/nrfconnect/sdk-zephyr/commit/2f39f35b1b7d1fbc1813606a435f410b7f74960b).